package.json

```diff
@@ -28,6 +28,7 @@
     "promise-pjs": "1.1.3",
     "prop-types": "15.7.2",
     "react-dates": "15.5.3",
+    "rrule": "2.6.2",
```

Comment:
Are we sure this is the correct version?

Reply:
Not sure. @cvializ Can you advise?

Reply:
Major version matches and they use semver. It should be OK if the tests pass

Reply:
Rolled back to v2.2.0 (which was originally checked in to `third_party/`)